src/app/shell/About.tsx

```diff
@@ -54,44 +83,89 @@ export default class About extends React.Component<Props> {
             </ExternalLink>
           </p>
         )}
-        <dl>
-          <dt>{t('Views.About.Twitter')}</dt>
-          <dd>
+        <div className="social">
+          <div>
+            <h2>
+              <ExternalLink href={openCollectiveLink}>
+                <AppIcon icon={faHeart} /> {t('Views.Support.Support')}
+              </ExternalLink>
+            </h2>
+            <div
+              dangerouslySetInnerHTML={{
+                __html: t('Views.Support.OpenCollective', { link: openCollectiveLink })
+              }}
+            />
+          </div>
+          <div>
+            <h2>
+              <ExternalLink href={openCollectiveLink}>
+                <AppIcon icon={faTshirt} /> {t('Header.Shop')}
+              </ExternalLink>
+            </h2>
+            <div
+              dangerouslySetInnerHTML={{
+                __html: t('Views.Support.Teespring', { link: storeLink })
```

Comment:
If we are going to change the value of `Views.Support.Teespring` we might as well just change the name of the key as well `Views.Support.Store`